pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -1,16 +1,19 @@
-{ stdenv, symlinkJoin, fetchurl, boost, brotli, cmake, flatbuffers, gtest, gflags, lz4, python, rapidjson, snappy, zlib, zstd }:
+{ stdenv, fetchpatch, symlinkJoin, fetchurl, boost, brotli, cmake, flatbuffers, gtest, gflags, lz4, python, rapidjson, snappy, zlib, zstd }:
```

Comment:
It looks like the patch is inlined and not actually retrieved via `fetchpatch`; leftover from development?

Reply:
Good catch! I had a private overlay with a bugfix that I never made a PR for. Now fixed.